### PR TITLE
Add separate route for getting the embed html and add support for all media in render endpoint

### PR DESCRIPTION
**Pull request by kavin-happyfox**
PR where the feature is utilised:: https://github.com/happyfoxinc/hf-editor/pull/65

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -1387,4 +1387,46 @@ export function isBlocked(url, options, cb) {
...
+            <p style="margin: 0 0 10px;">${errorMessage}</p>
+        </div>
+    </div>`;
+
+    return html;
+}
+
+export function getMediaEmbedWrapperHtml(oembedData) {
+    const { url, provider_name, html } = oembedData;
+    let paddingBottom = '56.25%'; // 16:9 aspect ratio, 9/16 = 0.5625
```
Comment (abishek-happyfox) [inline lib/utils.js:1420]: can we add like below 
`const paddingBottom = html ? '20%' : '56.25%';`

File: modules/api/views.js [hunk 2]
```diff
@@ -348,32 +356,56 @@ export default function(app) {
...
+                            if (oembed.html) {
+                                // Check if oembed has html
+                                render_link = {
+                                    html: oembed.html
+                                };
+                            } else {
+                                // Generate a default html based on oembed data
+                                render_link = {
+                                    html: iframelyUtils.generateDefaultHtmlFromMeta(oembed)
+                                };
```
Comment (abishek-happyfox) [inline modules/api/views.js:377]: can we simply do like below
```
render_link = {
          html: oembed.html || iframelyUtils.generateDefaultHtmlFromMeta(oembed)
 };
```

File: modules/api/views.js [hunk 3]
```diff
@@ -481,4 +513,54 @@ export default function(app) {
...
+                        const renderHtml = iframelyUtils.getMediaEmbedWrapperHtml(oembed);
+
+                        cb(error, renderHtml);
+                    });
+    
+                }, cb);
+            }
+    
+        ], function(error, renderHtml) {
+            if (error) {
```
Comment (abishek-happyfox) [inline modules/api/views.js:565]: can we add log for the error case if possibe?

**Review by abishek-happyfox — APPROVED** [review]

File: modules/api/views.js (hunk 3 above)
Comment (kavin-happyfox) [reply modules/api/views.js:565]: yes, but later

File: lib/utils.js (hunk 1 above)
Comment (kavin-happyfox) [reply lib/utils.js:1420]: since it will be expanded in future, lets keep it this way.

File: modules/api/views.js (hunk 2 above)
Comment (kavin-happyfox) [reply modules/api/views.js:377]: addressed

**Review by abishek-happyfox — APPROVED** [review]

**Pull request merged**